.babelrc

```diff
@@ -0,0 +1,18 @@
+{
+  "plugins": [
+    "transform-react-jsx",
+  ],
+  "presets": [
+    "es2015",
+    "react",
+    "stage-0"
+  ],
+  "env": {
+    "test": {
+      "plugins": "rewire"
+    },
```

Comment:
If we can, I think it would be good to avoid rewire. There are more widespread patterns to accomplish the same thing, e.g., jest mocks.

Reply:
Cool - this was mostly copy/pasta from `stitchfix/kufak_gtm_client`, so happy to get rid of it.

Reply:
Thanks! Rewire was in Knit and the Rails/Webpacker template a couple years ago, and still seems to find its way into our new apps now and again 😂.